content/git/index.md

```diff
@@ -78,17 +106,21 @@ Better Example:
 
 * Don't split a single *logical change* into several commits. For example,
   the implementation of a feature and the corresponding tests should be in the
-  same commit. Changes to a precompiled style resource and the resulting compiled `.css` should be in the same commit.
+  same commit.
```

Comment:
This seems to contradict slightly with the next bullet. I wonder if you should mention squashing here. I don't know if it's always clear what a "logical change" is. Often you will make multiple commits on a single feature, but then squash them for the PR.

Reply:
The more I think about this one, I disagree with the example. How about this:

> Commit early and often. Small, self-contained commits are easier to understand and revert when something goes wrong.

> That being said, try not to split a single logical change into multiple commits that will not function independently. If further consolidation of commits is desired before merging into the main branch, use [git rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing) or [git merge](https://git-scm.com/docs/git-merge#Documentation/git-merge.txt---squash) to squash commits.